internal/config/config.go

```diff
@@ -113,6 +117,20 @@ func Default() Config {
 	}
 }
 
+func ParseBinary(b Binary, kind string) error {
```

Comment:
Make this a method:
```suggestion
func (b *Binary) setup() error
```
And handle the errors individually: `fmt.Errorf("global: %w")`

Reply:
Solved